dlpy/applications/enet.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# Copyright SAS Institute
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from dlpy.model import Model
+from dlpy.sequential import Sequential
+from dlpy.layers import (Conv2d, Input, Pooling, BN, Conv2DTranspose, Concat, Res, Segmentation)
+from .application_utils import get_layer_options, input_layer_options
+
+
+def initialBlock(inp):
+    '''
+    Defines the initial block of ENet
+
+    Parameters
+    ----------
+    inp : Input layer
```

Comment:
variable_name : variable_type
its description

would you please update your documentation in the way above.

Reply:
Fixed.